packages/jest-console/src/__tests__/CustomConsole.test.ts

```diff
@@ -187,6 +187,15 @@ describe('CustomConsole', () => {
     });
   });
 
+  describe('dir', () => {
```

Comment:
Can we also test that it does not contain `depth` as a regression test?
In theory, Node could decide to print this object in full depth by default, and this test would pass even with the bug on master - but not if we check that `{depth:6}` is not printed to the console

Reply:
:+1: Good idea!